ui-frontend/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -101,12 +112,21 @@ module.exports = Backbone.AssociatedModel.extend({
       value = options
     }
     Backbone.AssociatedModel.prototype.set.call(this, key, value, options)
-    Common.queueExecution(() => {
```

Comment:
📝  This was removed in master, see: https://github.com/codice/ddf-ui/blob/master/ui-frontend/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js#L95-L102